packages/extension/src/worker/service-worker.ts

```diff
@@ -167,15 +167,3 @@ chrome.windows.onRemoved.addListener(async (windowId) => {
 chrome.runtime.onInstalled.addListener(async () => {
   await chrome.storage.local.clear();
 });
-
-/**
- * Listen to changes in cookie store and sync extension local store
- * when any cookies are delted ( through application tab in devtools )
- */
-chrome.cookies.onChanged.addListener(
-  async (changeInfo: chrome.cookies.CookieChangeInfo) => {
-    if (changeInfo.cause === 'explicit' && changeInfo.removed) {
-      await CookieStore.deleteCookie(changeInfo.cookie.name);
```

Comment:
@ayushnirwal How would we sync the cookie deletion without this? Have you thought of an alternative?

Reply:
Unfortunately no, but CDP might help. For now IMO we can delete this because a bug which is a blocker of v0.1 but only fixes a minor edge case.